web/server.ts

```diff
@@ -42,7 +44,13 @@ export const server = (async function setupWebserver() {
     });
 
     // Parse Cookies and JSON Bodies:
-    app.use(bodyParser.json());
+    app.use(
+        bodyParser.json({
+            verify: (req: WithRawBody<IncomingMessage>, res, buf) => {
+                req.rawBody = buf;
```

Comment:
A comment please why this is needed, implementing "verify" seems strange here on a first sight (but it makes sense to keep the raw buffer for the request)